### DISCUSSION TITLE
Error when flashing new boot image.

### DISCUSSION THREAD

**Discussion by TheDoctor0587** [ThomasPrior/LenovoSmartClock2#14] [Q&A]
Hi,

Instructions working great to the point where I flash the patched boot image.

I get the following error:

filename doesn't exists: vbmeta.img.empty, aborting flash write.

Any advice on the right course of action here?

Thanks!

**Comment by ThomasPrior** [comment]
What is the exact command you're issuing?

**Reply by TheDoctor0587** [reply]
Sorry, should've added that!

`python3 mtk.py w boot,vbmeta boot.patched,vbmeta.img.empty`

**Reply by TheDoctor0587** [reply]
Ran the command without reference to vbmeta and worked OK. Just had to re-setup the clock and reinstall the launcher.

**Reply by gdst4rp** [reply]
How did you solve this issue? I'm stuck at the same step, and if I try to boot the device afterwards, it get stuck at the Google setup step. I've since re-written the original partition files I backed up with mtk_gui.

**Reply by TheDoctor0587** [reply]
Mine didn't get stuck - Google Home added it all just fine. They'd never been previously connected though. Maybe worth removing all references to the clock from your Google Home app as I've found it's often more finicky to pair previously known devices sometimes. Sorry I can't give you more advice than that!

**Reply by gdst4rp** [reply]
Would you mind sharing the exact command you ran instead of "python3 mtk.py w boot,vbmeta boot.patched,vbmeta.img.empty"? Perhaps I'm misreading the next steps, but after all this, is it supposed to boot back into the Google initial setup? I couldn't seem to get to the next steps of updating the Webview components, and no USB access to run the adb commands.

Previously I had been running Last Launcher and using a Home Assistant dashboard via Fully Kiosk Browser, but of course that older webview and memory leaking reboots were becoming annoying.
